packages/flutter/lib/src/painting/continuous_rectangle_border.dart

```diff
@@ -3,58 +3,251 @@
+    //
+    // The shape is drawn from the top midpoint to the upper right hand corner
+    // in a clockwise fashion around to the upper left hand corner.
+    Path bezierRoundedRect () {
+      return Path()
+        ..moveTo(leftX(1.52866483), topY(0.0))
+        ..lineTo(rightX(1.52866471), topY(0.0))
+        ..cubicTo(rightX(1.08849323), topY(0.0),
+            rightX(0.86840689), topY(0.0),
+            rightX(0.66993427), topY(0.06549600))
+        ..lineTo(rightX(0.63149399), topY(0.07491100))
+        ..cubicTo(rightX(0.37282392), topY(0.16905899),
+            rightX(0.16906013), topY(0.37282401),
+            rightX(0.07491176), topY(0.63149399))
+        ..cubicTo(rightX(0), topY(0.86840701),
+            rightX(0.0), topY(1.08849299),
+            rightX(0.0), topY(1.52866483))
+        ..lineTo(rightX(0.0), bottomY(1.52866471))
+        ..cubicTo(rightX(0.0), bottomY(1.08849323),
+            rightX(0.0), bottomY(0.86840689),
+            rightX(0.06549600), bottomY(0.66993427))
+        ..lineTo(rightX(0.07491100), bottomY(0.63149399))
+        ..cubicTo(rightX(0.16905899), bottomY(0.37282392),
+            rightX(0.37282401), bottomY(0.16906013),
+            rightX(0.63149399), bottomY(0.07491176))
+        ..cubicTo(rightX(0.86840701), bottomY(0),
+            rightX(1.08849299), bottomY(0.0),
+            rightX(1.52866483), bottomY(0.0))
+        ..lineTo(leftX(1.52866483), bottomY(0.0))
+        ..cubicTo(leftX(1.08849323), bottomY(0.0),
+            leftX(0.86840689), bottomY(0.0),
+            leftX(0.66993427), bottomY(0.06549600))
+        ..lineTo(leftX(0.63149399), bottomY(0.07491100))
+        ..cubicTo(leftX(0.37282392), bottomY(0.16905899),
+            leftX(0.16906013), bottomY(0.37282401),
+            leftX(0.07491176), bottomY(0.63149399))
+        ..cubicTo(leftX(0), bottomY(0.86840701),
+            leftX(0.0), bottomY(1.08849299),
+            leftX(0.0), bottomY(1.52866483))
+        ..lineTo(leftX(0.0), topY(1.52866471))
+        ..cubicTo(leftX(0.0), topY(1.08849323),
+            leftX(0.0), topY(0.86840689),
+            leftX(0.06549600), topY(0.66993427))
+        ..lineTo(leftX(0.07491100), topY(0.63149399))
+        ..cubicTo(leftX(0.16905899), topY(0.37282392),
+            leftX(0.37282401), topY(0.16906013),
+            leftX(0.63149399), topY(0.07491176))
+        ..cubicTo(leftX(0.86840701), topY(0),
+            leftX(1.08849299), topY(0.0),
+            leftX(1.52866483), topY(0.0))
+        ..close();
+    }
+
+    // The ratio of the declared corner radius to the total affected pixels
+    // along each axis to render the corner. For example if the declared radius
+    // were 25px then totalAffectedCornerPixelRatio * 25 (~38) pixels would be
+    // affected along each axis.
+    //
+    // It is also the multiplier where the resulting shape will be convex with
+    // a height and width of any value. Below this value, noticeable clipping
```

Comment:
In general, be specific and avoid subjective words like noticeable and large. If they're important, quantify. Otherwise, just say adjacent corner transitions will intersect with each other and clip.